mm/2s2h/Rando/Logic/Regions/Central.cpp

```diff
@@ -18,6 +18,9 @@ static RegisterShipInitFunc initFunc([]() {
         .connections = {
             CONNECTION(RR_TERMINA_FIELD, CAN_GROW_BEAN_PLANT)
         },
+        .events = {
+            EVENT(RE_TERMINA_FIELD_SCRUB_ENTERED_GROTTO, true), // need Ocarina OR WEEKEVENTREG_74_40, which is set when getting RC_ASTRAL_OBSERVATORY_MOON_TEAR
```

Comment:
I think what we want to do here, for both this and the moon tear, is add an EVENT_ACCESS for being able to access the telescope in astral observatory. This should also allow it to work with entrance rando